Assets/Scripts/OldLady.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+public class OldLady : MonoBehaviour
+{
+    private float force_backwards = 500f;
+    private float force_upwards = 200f;
+
+    [SerializeField] AudioSource ladyTalking1; 
```

Comment:
You should have one audio source, and multiple audio clips instead. You can just load the clip you want into the audio source and play it